src/js/components/visualization/GroupSubMenu.jsx

```diff
@@ -72,12 +73,16 @@ var Group = connect(
     return iconMarkup;
   },
   deleteUpdate: function(id) {
+    ReactTooltip.hide();
     this.deleteMark(id);
     // set selected to null
     this.props.select(null);
     // redraw sidebar
     this.updateSidebar();
   },
+  componentWillUnmount: function(){
+    ReactTooltip.hide();
+  },
```

Comment:
If this is the failsafe, would it be sufficient to only call hide here, rather than in deleteUpdate?